modules/scripts/meshManager.js

```diff
@@ -1,120 +1,125 @@
-import * as THREE 				from './three.js';
-import { GLTFLoader } 		from './GLTFLoader.min.js';
+import * as THREE from './three.js';
+import { GLTFLoader } from './GLTFLoader.min.js';
 
-const Meshes = [
-/* Board Components */
+const Meshes = [
+	/* Board Components */
 	{
-		n:'sea',
-		s:0.8,
-		a:false,
+		n: 'sea',
+		s: 0.8,
+		a: false,
 	},
 	{
-		n:'island',
-		s:6.5,
-		a:false,
+		n: 'island',
+		s: 6.5,
+		a: false,
 	},
 	{
-		n:'board',
-		t:'island',
-		s:0.84,
-		a:false,
+		n: 'board',
+		t: 'island',
+		s: 0.84,
+		a: false,
 	},
 	{
-		n:'outerWall',
-		t:'island',
-		s:0.84,
-		a:false,
+		n: 'outerWall',
+		t: 'island',
+		s: 0.84,
+		a: false,
 	},
 	{
-		n:'innerWall',
-		t:'island',
-		s:0.84,
-		a:false,
+		n: 'innerWall',
+		t: 'island',
+		s: 0.84,
+		a: false,
 	},
 
-/* Lvl */
+	/* Buildings */
 	{
-		n:'lvl0',
-		s:0.32,
-		l:0x89968a,
+		n: 'lvl0',
+		s: 0.32,
+		l: 0x89968a,
 	},
 	{
-		n:'lvl1',
-		s:0.32,
-		l:0x89968a,
+		n: 'lvl1',
+		s: 0.32,
+		l: 0x89968a,
 	},
 	{
-		n:'lvl2',
-		s:0.32,
-		l:0x89968a,
+		n: 'lvl2',
+		s: 0.32,
+		l: 0x89968a,
 	},
 	{
-		n:'lvl3',
-		s:0.32,
+		n: 'lvl3',
+		s: 0.32,
 	},
 
 
-/* Workers */
+	/* Workers */
 	{
-		n:'f0worker',
-		g:'fWorker',
-		s:0.9,
+		n: 'f0worker',
+		g: 'fWorker',
+		s: 0.9,
 	},
 	{
-		n:'m0worker',
-		g:'mWorker',
-		s:0.9
+		n: 'm0worker',
+		g: 'mWorker',
+		s: 0.9
 	},
 	{
-		n:'f1worker',
-		g:'fWorker',
-		s:0.9
+		n: 'f1worker',
+		g: 'fWorker',
+		s: 0.9
 	},
 	{
-		n:'m1worker',
-		g:'mWorker',
-		s:0.9
+		n: 'm1worker',
+		g: 'mWorker',
+		s: 0.9
 	},
 	{
-		n:'f2worker',
-		g:'fWorker',
-		s:0.9
+		n: 'f2worker',
+		g: 'fWorker',
+		s: 0.9
 	},
 	{
-		n:'m2worker',
-		g:'mWorker',
-		s:0.9
+		n: 'm2worker',
+		g: 'mWorker',
+		s: 0.9
 	},
 	{
-		n:'ram',
-		s:0.42,
+		n: 'ram',
+		s: 0.42,
 	},
 
-
+	/* Tokens */
+	{
+		n: 'tokenWhirpool',
+		g: 'token',
+		s: 0.8,
+	},
 	{
-		n:'tokenWhirpool',
-		g:'token',
-		s:0.8,
+		n: 'tokenTalus',
+		g: 'token',
+		s: 0.9,
 	},
 	{
-		n:'tokenTalus',
-		g:'token',
-		s:0.8,
+		n: 'tokenCoin',
```

Comment:
Only actual change in meshManager.js is adding new tokenCoin, the rest was just whitespace formatting done by VSCode.